line_detection_updated.py

```diff
@@ -0,0 +1,302 @@
+'''
+This module creates the functions and class that are used to detect
+straight lines in astronomical images.
+
+Process
+---------
+1. Ingest the raw image
+2. Apply brightness cuts and standardize the image
+3. Perform binary thresholding on the image
+4. Blur the image
+5. Find edges
+6. Apply Hough line on the edge image
+7. Cluster the estimated lines
+8. Return all the estimated lines along with their cluster label
+
+'''
+
+# Importing necessary libraries
+import numpy as np
+import matplotlib.pyplot as plt
+
+import cv2
+
+from skimage.transform import hough_line, hough_line_peaks
+from sklearn.cluster import MeanShift
+
+# Defining functions
+
+
```

Comment:
Flake 8 won't complain about this as it's technically correct but this space here can be trimmed up. On a broader note the comments are only really useful if they are revealing something not-obvious about the code they are commenting, when you have a situation like:
```
# calculate error
def calculate_error()....
```
the comment is not particularly helpful. Just as a heads up for future use.